README.md

```diff
@@ -411,6 +411,7 @@ Passes additional driver-specific options. Details for each driver:
   - `image=IMAGE` - Sets the container image to be used for running buildkit.
   - `namespace=NS` - Sets the Kubernetes namespace. Defaults to the current namespace.
   - `replicas=N` - Sets the number of `Pod` replicas. Defaults to 1.
+  - `nodeSelector=label1=value1&label2=value2` - Sets the kv of `Pod` nodeSelector. No Defaults. Example `nodeSelector=kubernetes.io/arch=arm64`
```

Comment:
We don't use camelCase here. node-selector or just nodeselector?

Reply:
Also curious why "&"?

Reply:
sure, i will change this.

nodeSelector is `map[string]string` of k8s

i used url.ParseQuery to parse it, this why `&` here.

Reply:
Comma is the kube standard https://godoc.org/k8s.io/apimachinery/pkg/fields#ParseSelector

Reply:
but is may conflict with driver opts

`replicas=3,nodeselector=kubernetes.io/arch=arm64,kubernetes.io/os=linux`

`replicas=3,nodeselector="kubernetes.io/arch=arm64,kubernetes.io/os=linux"`

which one will be correct?

Reply:
The latter one